.github/workflows/release-test.yml

```diff
@@ -5,12 +5,30 @@ on:
     branches:
       - master
     paths-ignore:
+      - '.github/**'
+      - '.git/**'
+      - '.gitattributes'
+      - '.idea/**'
+      - '.vscode/**'
```

Comment:
I don't think we need to ignore `.idea` or `.vscode` here since `gitignore` has already done this for us

Reply:
@Yiyiyimu I've removed the paths that are already in `.gitignore`. `.vscode` is remained because it's not in it.

Reply:
Maybe adding `.vscode` to `.gitignore` would be better, since it's the same type as `.idea`